opennlp-tools/src/test/java/opennlp/tools/eval/AbstractEvalTest.java

```diff
@@ -18,73 +18,65 @@
 package opennlp.tools.eval;
 
 import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
 import java.math.BigInteger;
-import java.nio.file.Files;
-import java.nio.file.Path;
+import java.nio.charset.StandardCharsets;
 import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
 
 import org.junit.Assert;
 
 import opennlp.tools.ml.maxent.quasinewton.QNTrainer;
 import opennlp.tools.ml.naivebayes.NaiveBayesTrainer;
 import opennlp.tools.ml.perceptron.PerceptronTrainer;
+import opennlp.tools.util.ObjectStream;
 import opennlp.tools.util.TrainingParameters;
 import opennlp.tools.util.model.ModelUtil;
 
-public class EvalUtil {
+public abstract class AbstractEvalTest {
 
-  static final double ACCURACY_DELTA = 0.0001d;
+  public static final double ACCURACY_DELTA = 0.0001d;
+  public static final String HASH_ALGORITHM = "MD5";
 
-  static TrainingParameters createPerceptronParams() {
+  public static void verifyTrainingData(ObjectStream<?> samples, BigInteger checksum) throws Exception {
```

Comment:
I think the way it was done before - by looking at the bytes of the files - is better than looking at the samples, because sometimes there are changes in the way a format is parsed, and then this training data test has to be updated. We can add a second method which can do this conveniently for a directory.